Ginger/GingerCoreNET/Run/GingerExecutionEngine.cs

```diff
@@ -554,7 +554,7 @@ public void RunRunner(bool doContinueRun = false)
                     if (!mStopRun)//not on stop run
                     {
                         CloseAgents();
-                        if (mGingerRunner.ProjEnvironment != null)
+                        if (mGingerRunner.ProjEnvironment != null && standAloneRunnerExecution)
```

Comment:
Enviroment will be closed at the end of runset execution if standAloneRunnerExecution is set to false?

Reply:
if individual runner executed then, standAloneRunnerExecution will be true and Env will be closed at end of runner

if runset executed then, standAloneRunnerExecution will be false and Env will be closed at end of runset (RunsetExecutor.cs:369)

Reply:
Made the suggested changes with eRunLevel enum